packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -90,7 +90,7 @@ export const VAutocomplete = genericComponent<new <
     search: String,
 
     ...makeFilterProps({ filterKeys: ['title'] }),
-    ...makeSelectProps(),
+    ...makeSelectProps({ multiple: false, returnObject: false }),
```

Comment:
Why'd you add this? Boolean props already default to false.

Reply:
Thanks! That was something I didn't consider thoroughly. Removed.